TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/sockets/BungeeStringHandler.java

```diff
@@ -1,86 +1,38 @@
-                break;
-            case EVENT_FIRED:
-                try {
-                    EventType eventType = EnumUtil.valueOf(EventType.class, (String) message.get("eT"));
-                    ((EventManager) TimoCloudAPI.getEventAPI()).callEvent(((TimoCloudUniversalAPIBasicImplementation) TimoCloudAPI.getUniversalAPI()).getObjectMapper().readValue((String) data, EventUtil.getClassByEventType(eventType)));
-                } catch (Exception e) {
-                    System.err.println("Error while parsing event from json: ");
-                    TimoCloudBungee.getInstance().severe(e);
-                }
-                break;
-            case CORE_SEND_MESSAGE_TO_COMMAND_SENDER: {
-                TimoCloudBungee.getInstance().getTimoCloudCommand().sendMessage((String) message.get("sender"), (String) data);
-            }
-            case PROXY_EXECUTE_COMMAND:
-                TimoCloudBungee.getInstance().getProxy().getPluginManager().dispatchCommand(TimoCloudBungee.getInstance().getProxy().getConsole(), (String) data);
-                break;
-            case PROXY_ADD_SERVER:
-                TimoCloudBungee.getInstance().getProxy().getServers().put(server, TimoCloudBungee.getInstance().getProxy().constructServerInfo(server, new InetSocketAddress((String) message.get("address"), ((Number) message.get("port")).intValue()), "", false));
-                break;
-            case PROXY_REMOVE_SERVER:
-                TimoCloudBungee.getInstance().getProxy().getServers().remove(server);
-                break;
-            case CORD_SET_IP:
-                try {
-                    TimoCloudBungee.getInstance().getIpManager().setAddresses(
-                            InetAddressUtil.getSocketAddressByName((String) message.get("CHANNEL_ADDRESS")),
-                            InetAddressUtil.getSocketAddressByName((String) message.get("CLIENT_ADDRESS")));
-                } catch (Exception e) {
-                    TimoCloudBungee.getInstance().severe("Error while parsing IP addresses (" + message.get("CHANNEL_ADDRESS") + ", " + message.get("CLIENT_ADDRESS") + "): ");
-                    TimoCloudBungee.getInstance().severe(e);
-                }
-                break;
-            case ON_PLUGIN_MESSAGE: {
-                AddressedPluginMessage addressedPluginMessage = PluginMessageSerializer.deserialize((Map) data);
-                ((TimoCloudMessageAPIBasicImplementation) TimoCloudAPI.getMessageAPI()).onMessage(addressedPluginMessage);
-                break;
-            }
-            case PROXY_STOP: {
-                TimoCloudBungee.getInstance().stop();
-                break;
-            }
-            default:
-                TimoCloudBungee.getInstance().severe("Could not categorize json message: " + message);
-        }
+        getMessageHandlers(type).forEach(messageHandler -> messageHandler.execute(message, channel));
```

Comment:
We need some kind of exception handling so that the execution does not stop if one of the handlers throws an exception